.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - 8
+cache:
+  yarn: true
+  directories:
+    - node_modules
+script:
+  - cp src/containers/ApiKey.js.sample src/containers/ApiKey.js
+  - yarn test
```

Comment:
Do we need yarn build here? Or will it auto run build?

Reply:
No build step required here (no server dependency). Once we add deployment, we will call `yarn build` after the call to `yarn test`